web-registry/src/pages/CreateCreditClass/CreateCreditClass.tsx

```diff
@@ -1,251 +1,37 @@
-import React, { useState } from 'react';
-import { makeStyles, useTheme } from '@mui/styles';
-import useMediaQuery from '@mui/material/useMediaQuery';
-import CardMedia from '@mui/material/CardMedia';
-import Box from '@mui/material/Box';
-import Grid from '@mui/material/Grid';
-
-import Section from 'web-components/lib/components/section';
-import ResponsiveSlider from 'web-components/lib/components/sliders/ResponsiveSlider';
-import FixedFooter from 'web-components/lib/components/fixed-footer';
-import ContainedButton from 'web-components/lib/components/buttons/ContainedButton';
-import Modal from 'web-components/lib/components/modal';
-import { Body, Title } from 'web-components/lib/components/typography';
-import { BlockContent } from 'web-components/lib/components/block-content';
+import { Box } from 'web-components/lib/components/box';
 
+import { MultiStepTemplate } from 'components/templates/MultiStepTemplate';
+import { useWallet } from 'lib/wallet';
```

Comment:
this large diff is because the component was repurposed... prior component now lives in `CreateCredditClassInfo.tsx`